### PR TITLE
Editor track selection: Audio only

### PR DESCRIPTION
**Pull request by Arnei**
*Work in progress*

The backend pendant to https://github.com/opencast/opencast-editor/pull/1230.

This is also very interested in https://github.com/opencast/opencast/pull/5856

### CODE REVIEW

**Comment by mtneug** [comment]
Note: This PR is targeting the legacy branch of Opencast (`r/15.x`). In [a recently passed proposal](https://groups.google.com/a/opencast.org/g/dev/c/VwYqk3vXdd0), we changed how we deal with PRs to legacy branches.

Our goal is to provide stability and minimize the risk of breaking changes. Ultimately, users should trust that patch releases of legacy versions can be installed easily.

To this end, developers targeting legacy branches now need to explain why those changes are necessary and can't just go into stable or develop. We'd appreciate it if you could provide such an explanation or, alternatively, retarget this PR to stable or develop.

File: modules/editor-service/src/main/java/org/opencastproject/editor/EditorServiceImpl.java [hunk 1]
```diff
@@ -1042,9 +1042,15 @@ public EditingData getEditData(final String mediaPackageId) throws EditorService
 
     User user = securityService.getUser();
 
+    // TODO
+    Boolean customizedTrackSelection = false;
+    if (latestWfProperties.get("hide_tracks").equals("true")) {
```
Comment (marwyg) [inline modules/editor-service/src/main/java/org/opencastproject/editor/EditorServiceImpl.java:1047]: `if (latestWfProperties.get("hide_tracks") != null && latestWfProperties.get("hide_tracks").equals("true")) {`

(Nullpointer Fix)

**Comment by Arnei** [comment]
The recent release of Opencast 15.12 is intended to be the final minor release for 15. Please retarget your PR or explain why it warrants another Opencast 15 release.

**Comment by Arnei** [comment]
As work has been halted indefinitely, I'm closing this for now.

**Pull request closed without merging**